addons/addon-base-raas/packages/base-raas-rest-api/lib/controllers/aws-accounts-controller.js

```diff
@@ -36,6 +37,19 @@ async function configure(context) {
     }),
   );
 
+  // ===============================================================
+  //  GET /permissions (mounted to /api/aws-accounts)
+  // ===============================================================
+  router.get(
+    '/permissions',
+    wrap(async (req, res) => {
+      const requestContext = res.locals.requestContext;
+      const accountsList = await awsAccountsService.list();
+      const result = await awsCfnService.batchCheckAccountPermissions(requestContext, accountsList);
```

Comment:
The call to awsAccountsService.list() should be part of awsCfnService instead of the router, to maintain separation of responsibility.

Reply:
okay, that makes sense. I was basing it off the /api/aws-accounts call, but now that I'm looking at it it's a bit of a different scenario. I'll make that change with the next commit.